src/api.rs

```diff
@@ -1082,59 +1082,66 @@ pub async fn gg18_sign_client_round9(context: String, delay: u32) -> Result<Stri
     let context = serde_json::from_str::<GG18SignClientContext>(&context)?;
     let client = new_client_with_headers()?;
     //////////////////////////////////////////////////////////////////////////////
-    broadcast(
-        &client,
-        &context.addr,
-        context.party_num_int,
-        "round9",
-        serde_json::to_string(&context.s_i.as_ref().unwrap())?,
-        context.uuid.clone(),
-    )
-    .await?;
-    let round9_ans_vec = poll_for_broadcasts(
-        &client,
-        &context.addr,
-        context.party_num_int,
-        context.threshould + 1,
-        "round9",
-        context.uuid.clone(),
-        delay,
-    )
-    .await?;
+    if context.party_id != 1 {
```

Comment:
here we assume the key owner's party_id is 1,  is that always true?  any graceful way to detect the owner?